packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -216,6 +216,9 @@ async function nodeImport(
   ) => {
     const resolved = tryNodeResolve(id, importer, options, false)
```

Comment:
Note that `tryNodeResolve` runs first, but won't return a match. 

The problematic `id` is already a fully resolved, absolute path to a file that does exist... maybe this needs to be fixed inside of `tryNodeResolve`?

Reply:
> maybe this needs to be fixed inside of `tryNodeResolve`?

Vite's internal resolve plugin checks for absolute paths before `tryNodeResolve`. See here:
https://github.com/vitejs/vite/blob/b45f4ad9f1336d1e88d271d7aca9498dde2e5013/packages/vite/src/node/plugins/resolve.ts#L195-L199